docs/source/en/model_doc/clvp.md

```diff
@@ -0,0 +1,85 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# CLVP
+
+## Overview
+
+The CLVP(Contrastive Language-Voice Pretrained Transformer) model was proposed in [Better speech synthesis through scaling](https://arxiv.org/abs/2305.07243) by James Betker.
+
+The abstract from the paper is the following:
+
+*In recent years, the field of image generation has been revolutionized by the appli-
+cation of autoregressive transformers and DDPMs. These approaches model the
+process of image generation as a step-wise probabilistic processes and leverage
+large amounts of compute and data to learn the image distribution.
+This methodology of improving performance need not be confined to images. This
+paper describes a way to apply advances in the image generative domain to speech
+synthesis. The result is TorToise - an expressive, multi-voice text-to-speech sys-
+tem.*
+
+Tips:
+
+1. CLVP or Contrastive Language-Voice Pretrained Transformer is an integral part of the `tortoise-tts` system.
+2. In `tortoise-tts`, CLVP is used to compare different generated speech token candidates (generated by the 
+auto-regressive transformer model) with the provided text, and the best speech tokens are forwarded to the diffuser model.
+3. Since the conversion of text to text-tokens is done by `tortoise-tts` tokenizer, we don't have any tokenizer implemented
```

Comment:
Hmmm don't agree with the design here - if we're adding the model to `transformers` we should be able to use it as a standalone model (i.e. with its own tokenizer)

Is the CLVP tokenizer the same as any other tokenizer in the library? If it's one-to-one the same as an existing tokenizer, we don't need a new `tokenizing_clvp.py` file, and can just load the tokenizer using `AutoTokenizer.from_pretrained(...)`. Otherwise, we need to add the tokenizer in its own new tokenizer file